app/views/layouts/_right_section.html.haml

```diff
@@ -0,0 +1 @@
+= react('RightSection', { :customLogo => ::Settings.server.custom_logo, :currentUser => current_user, :helpMenu => menu_to_json(:help),:opsExplorerAllowed => ApplicationHelper.role_allows?(:feature => 'ops_explorer', :any => true), :applianceName => appliance_name, :miqGroups => current_user.miq_groups, :currentGroup => current_group, :userMenu => menu_to_json(:top_right) }, {:class => "nav navbar-nav navbar-right navbar-iconic"}, 'ul')
```

Comment:
Missing newline

Reply:
Maybe it's completely wrong and maybe out of the context of this PR, but could we preseed these data into the Redux store upon login? Then we would have them cached and the list of props would be simpler/shorter/nicer...